tox.ini

```diff
@@ -0,0 +1,27 @@
+[tox]
+envlist = py{27,34,35}-django{18,19}
+
+[testenv]
+deps =
+    -rdocs/requirements.txt
+    jinja2
+    https://github.com/tBaxter/vobject/tarball/master#egg=vobject
```

Comment:
Why the Jinja2 dependency?

Also, can we scrub references to vobject and replace with python-card-me (replaces vobject)?